mlos_bench/mlos_bench/optimizers/base_optimizer.py

```diff
@@ -242,16 +242,24 @@ def bulk_register(self,
         is_not_empty : bool
             True if there is data to register, false otherwise.
         """
-        _LOG.info("Update the optimizer with: %d configs, %d scores, %d status values",
-                  len(configs or []), len(scores or []), len(status or []))
+        _LOG.info(
+            "Update the optimizer with: %d configs, %d scores, %d status values",
+            len(configs or []), len(scores or []), len(status or [])
+        )
+
         if len(configs or []) != len(scores or []):
             raise ValueError("Numbers of configs and scores do not match.")
         if status is not None and len(configs or []) != len(status or []):
             raise ValueError("Numbers of configs and status values do not match.")
+
         has_data = bool(configs and scores)
-        if has_data and self._start_with_defaults:
+        if len(configs or []) == 1:
+            _LOG.info("Only one configuration provided, using defaults.")
+            self._start_with_defaults = True
+        elif has_data and self._start_with_defaults:
```

Comment:
This seems like the meat of the change.  Can you please update the PR description to explain this?

Reply:
If there was no config it would freak out and say hey you have no trials in the db I cannot suggest a config based on nothing